### PR TITLE
Unified History: Place the `UnifiedHistory` icon

### LINKED ISSUE #97961: Unified History: Determine where in the product this feature should live
Place the `HistoryContainer` on the right hind of `CommandPalette` on the topNav.

### PR DESCRIPTION
**Pull request by eledobleefe**
**What is this feature?**
We move the icon to match the designs and modify the grid to adapt to the new length of elements to show on the right hand of the TopNav.

**Why do we need this feature?**
To match the designs.

**Who is this feature for?**
Everybody interested in `UnifiedHistory`.

**Which issue(s) does this PR fix?**:

Fixes #97961

**Special notes for your reviewer:**

Please check that:
- [ ] It works as expected from a user's perspective.
- [ ] If this is a pre-GA feature, it is behind a feature toggle.
- [ ] The docs are updated, and if this is a [notable improvement](https://grafana.com/docs/writers-toolkit/contribute/release-notes/#how-to-determine-if-content-belongs-in-whats-new), it's added to our [What's New](https://grafana.com/docs/writers-toolkit/contribute/release-notes/) doc.

### CODE REVIEW

File: public/app/core/components/AppChrome/TopBar/SingleTopBar.tsx [hunk 1]
```diff
@@ -73,6 +73,7 @@ export const SingleTopBar = memo(function SingleTopBar({
 
       <Stack gap={0.5} alignItems="center">
         <TopSearchBarCommandPaletteTrigger />
+        {unifiedHistoryEnabled && <HistoryContainer />}
```
Comment (ashharrison90) [inline public/app/core/components/AppChrome/TopBar/SingleTopBar.tsx:76]: this is here twice now 😬

File: public/app/core/components/AppChrome/TopBar/SingleTopBar.tsx [hunk 2]
```diff
@@ -115,11 +116,20 @@ const getStyles = (theme: GrafanaTheme2, menuDockedAndOpen: boolean) => ({
     justifyContent: 'space-between',
 
     [theme.breakpoints.up('lg')]: {
-      gridTemplateColumns: '2fr minmax(440px, 1fr)',
+      gridTemplateColumns: '2fr minmax(50%, 1fr)',
```
Comment (ashharrison90) [inline public/app/core/components/AppChrome/TopBar/SingleTopBar.tsx:119]: i'm not sure this should be a percentage, doesn't this kind of break the `minmax`? like the min (`50%`) is always going to be higher than the max (`1fr`)

File: public/app/core/components/AppChrome/TopBar/SingleTopBar.tsx [hunk 3]
```diff
@@ -115,11 +116,20 @@ const getStyles = (theme: GrafanaTheme2, menuDockedAndOpen: boolean) => ({
...
+        gridTemplateColumns: '2fr minmax(55%, 1fr)',
+      }),
+    },
+    [theme.breakpoints.up('xl')]: {
+      gridTemplateColumns: '2fr minmax(40%, 1fr)',
+      ...(menuDockedAndOpen && {
+        gridTemplateColumns: '2fr minmax(55%, 1fr)',
+      }),
     },
+    [theme.breakpoints.up('xxl')]: menuDockedAndOpen && { gridTemplateColumns: '2fr minmax(40%, 1fr)' },
```
Comment (ashharrison90) [inline public/app/core/components/AppChrome/TopBar/SingleTopBar.tsx:132]: i don't think we should have this many different breakpoint styles... is this just a side effect of using a percentage in the first case? 🤔

**Review by ashharrison90 — COMMENTED** [review]
nits

File: public/app/core/components/AppChrome/TopBar/SingleTopBar.tsx (hunk 3 above)
Comment (eledobleefe) [reply public/app/core/components/AppChrome/TopBar/SingleTopBar.tsx]: Yeah, I reverted the changes and just increased a bit the minimum. If needed, we can always iterate on this in the future.

**Review by JoaoSilvaGrafana — APPROVED** [review]
Lgtm! 👏

**Pull request merged**